person.rb

```diff
@@ -2,22 +2,25 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :classroom, :rentals
```

Comment:
- [ ] Hello there! 🤗 Please make sure to include a setter for the @rental variable. This is important because we'll need to not only read but also make modifications to it. 📝✏️ By adding a setter, we can ensure that the @rental variable becomes more flexible and adaptable, allowing us to make the necessary changes as required. 🔧💼
- [ ] Since the relationship association primarily pertains to students and classrooms, there's no need to include getters for classrooms in the Person class. 🚫

```suggestion
 attr_accessor :name, :age, :rentals
 attr_reader :id
```